{{cookiecutter.project_slug}}/Makefile

```diff
@@ -21,10 +21,10 @@ REQUIREMENTS_APT_ENV?=$(PROJECT_PATH_ENV)/$(REQUIREMENTS_APT)
 RESULTS_PATH_ENV?=$(PROJECT_PATH_ENV)/$(RESULTS_PATH)
 
 SETUP_NAME?=setup
-TRAINING_NAME?=training
-JUPYTER_NAME?=jupyter
-TENSORBOARD_NAME?=tensorboard
-FILEBROWSER_NAME?=filebrowser
+TRAINING_NAME?=training-{{cookiecutter.project_slug}}
```

Comment:
I think we may use the same idea for setup, it may take some time as well.

Reply:
ok, sounds good